components/function-controller/internal/controllers/serverless/gitops.go

```diff
@@ -3,11 +3,21 @@ package serverless
 import (
 	"fmt"
 
-	ctrl "sigs.k8s.io/controller-runtime"
-
 	"github.com/kyma-project/kyma/components/function-controller/internal/git"
+	serverlessv1alpha2 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha2"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+//TODO refactor - move to system state
+func (r *FunctionReconciler) syncRevision(instance *serverlessv1alpha2.Function, options git.Options) (string, error) {
```

Comment:
Is this func used anywhere?

Reply:
Not used, a leftover, will delete, many thx